### ISSUE TITLE
Product images not created when using populatedb 

### ISSUE THREAD

**Issue by matemijolovic** [mirumee/saleor#3548]
### What I'm trying to achieve
I'm trying to run a clean install of Saleor using Docker by following the [documentation](https://saleor.readthedocs.io/en/latest/customization/docker.html) and populate the database with the example data using _populatedb_. 

After the process has finished, the product images for example products are missing.

The directory /app/media in container saleor_web is empty. 

Celery log shows messages like the following for every product image:

```
[2019-01-04 10:33:41,318: ERROR/ForkPoolWorker-7] Thumbnail generation failed

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/versatileimagefield/image_warmer.py", line 117, in _prewarm_versatileimagefield
    url = get_url_from_image_key(versatileimagefieldfile, size_key)
  File "/usr/local/lib/python3.6/site-packages/versatileimagefield/utils.py", line 216, in get_url_from_image_key
    img_url = img_url[size_key].url
  File "/usr/local/lib/python3.6/site-packages/versatileimagefield/datastructures/sizedimage.py", line 149, in __getitem__
    height=height
  File "/usr/local/lib/python3.6/site-packages/versatileimagefield/datastructures/sizedimage.py", line 201, in create_resized_image
    path_to_image
  File "/usr/local/lib/python3.6/site-packages/versatileimagefield/datastructures/base.py", line 140, in retrieve_image
    image = self.storage.open(path_to_image, 'rb')
  File "/usr/local/lib/python3.6/site-packages/django/core/files/storage.py", line 33, in open
    return self._open(name, mode)
  File "/usr/local/lib/python3.6/site-packages/django/core/files/storage.py", line 218, in _open
    return File(open(self.path(name), mode))

FileNotFoundError: [Errno 2] No such file or directory: '/app/media/category-backgrounds/DEMO-04.jpg'
```

### Steps to reproduce the problem
1. Clone saleor master branch 
2. Run steps described in the [documentation](https://saleor.readthedocs.io/en/latest/customization/docker.html)

### Screenshots
![Saleor error in dashboard](https://i.imgur.com/aWAJmCj.png) 

**System information**
Operating system: Ubuntu 18.04 LTS
Docker version 18.09.0
Docker-compose version 1.23.2

**Comment by hairychris** [comment]
This may caused by a folder permissions issue to do with running under docker, this pull request is somewhat related: https://github.com/mirumee/saleor/pull/3117

Another alternative is if you've been starting and stopping lots of containers lately you might have run into a loop of some kind with background tasks, this commit is also related:

https://github.com/mirumee/saleor/pull/3118/commits/48a6bc8bfeac4fe2c2bb7d11c04117efd9f5159e

**Comment by matemijolovic** [comment]
> This may caused by a folder permissions issue to do with running under docker, this pull request is somewhat related: #3117
> 
> Another alternative is if you've been starting and stopping lots of containers lately you might have run into a loop of some kind with background tasks, this commit is also related: [48a6bc8](https://github.com/mirumee/saleor/commit/48a6bc8bfeac4fe2c2bb7d11c04117efd9f5159e)

Thanks for the reply.

What I've tried so far:
- running migrations, db population and static collections with *--rm* flag, as described [here](https://github.com/mirumee/saleor/commit/48a6bc8bfeac4fe2c2bb7d11c04117efd9f5159e)
- tweaking the file permissions / running the containers as root
- running without the `--dev` flag in `pipenv install` command (this shouldn't matter either)
- pruning the docker system (including containers, images, volumes and networks) and starting fresh

Unfortunately, still no success. Will try to investigate further.

**Comment by hairychris** [comment]
For some reason the Saleor guys aren't interested in any fixes involving Docker it seems. Not sure if they just don't use it in development or whatever. Everything works in production so perhaps that's all they need to work. It kinda sucks because now there are loads of incompatible unmerged Docker related pull requests.

Another user proposed a fix but it wasn't accepted, they have created a seperate module with their fixes that you can install and might help you out with this: https://github.com/bennylb/saleor-docker-local

**Comment by matemijolovic** [comment]
Thanks for the replies. Meanwhile we managed to find a workaround by setting the shared media volume for `web` and `celery` containers.

**Comment by hairychris** [comment]
That sounds great thanks for the reply, I'll keep an eye out and help out where I can. It might also be worth investigating https://github.com/stephenmoloney/saleor-helm and seeing if it can / should be upstreamed.

**Comment by stephenmoloney** [comment]
I think the issue @matemijolovic relates to docker-compose rather than the Dockerfile. 
I have basically simplified the Dockerfile [here](https://github.com/stephenmoloney/saleor-helm/blob/master/Dockerfile) without making many changes.

I would propose creating a `saleor/helm` separate repo  instead of merging into the core repo. I'd also propose making a helm https charts server. I'm willing to collaborate if there is an interest on saleor side. 

Core repo is already overburdened with issues and helm/kubernetes issues would just add to the noise.

**Comment by maarcingebala** [comment]
Should be resolved by #3657.
